StonksWeb/Finances.aspx.cs

```diff
@@ -22,40 +22,57 @@ protected void Page_Init(object sender, EventArgs e)
             {
                 { ExpenseType.Housing, TextBoxHousing },
                 { ExpenseType.Groceries, TextBoxGroceries },
-                { ExpenseType.Transport, TextBoxTransportation },
+                { ExpenseType.Transportation, TextBoxTransportation },
                 { ExpenseType.Entertainment, TextBoxEntertainment },
-                { ExpenseType.Shopping, TextBoxShopping },
                 { ExpenseType.Health, TextBoxHealth },
+                { ExpenseType.Shopping, TextBoxShopping },
                 { ExpenseType.Utilities, TextBoxUtilities },
                 { ExpenseType.Other, TextBoxOther }
             };
         }
+
         protected void Page_LoadComplete(object sender, EventArgs e)
         {
-            TextBoxIncome.Text = Global.financialPlan.Income.ToString();
+            TextBoxIncome.Text = FinancialPlanController.ActivePlan.Income.ToString();
             foreach (KeyValuePair<ExpenseType, TextBox> boxType in boxTypeList)
             {
-                var expense = Global.financialPlan.GetExpense(boxType.Key);
+                var expense = FinancialPlanController.ActivePlan.GetExpense(boxType.Key);
                 if (expense != null)
                 {
                     boxType.Value.Text = expense.Value.ToString();
                 }
             }
         }
 
+        public void LoadExpenses()
+        {
+            //Income
+            if (TextBoxIncome == null)
```

Comment:
TextBoxIncome == null wont have the Text property. This if/else basically does nothing. If TextBoxIncome is not initialized the function shouldn't be called, but we can throw an exception if a mistake happens.